examples/vanilla/control-elements/media-rendition-listbox.html

```diff
@@ -53,10 +53,17 @@ <h1>Rendition Listbox</h1>
     Quality: <span id="renditionactive">N/A</span>
   </p>
 
-  <script>
+  <script type="module">
     video.addEventListener('resize', () => {
       renditionactive.textContent = `${Math.min(video.videoWidth, video.videoHeight)}p`;
     });
+
+    const RenditionListbox = await customElements.whenDefined('media-rendition-listbox');
+
+    RenditionListbox.formatOptionText = (text) => {
+      console.log(text);
+      return text.replace('720p', '720p (HD)');
+    }
```

Comment:
I chose to put this callback on the Listbox class because we talked before to use the native selectmenu if it would be supported by browsers and the selectmenu and option classes would not come with such a format callback.